pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md

```diff
@@ -176,6 +176,24 @@ Mostly credential issues.
 #### Solutions
 Check for your bot credentials.
 
+### com.liveperson.bot-connectors-worker.error.aivendor.dialogflowcx.parse-message.parser-error
+
+#### Description
+Testing connection with DialogFlow CX failed.
+#### Reasons
+Mostly credential issues.
+#### Solutions
+Check for your bot credentials.
```

Comment:
Verify the credentials provided for the bot.

Reply:
Done